pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "qiling"
+version = "1.4.7"
+description = "Qiling is an advanced binary emulation framework that cross-platform-architecture"
+authors = ["KaiJern Lau (xwings) <info@qiling.io>"]
+license = "GPLv2"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+capstone = "^4"
+unicorn = "^2"
+pefile = ">=2022.5.30"
+python-registry = "^1.3.1"
+keystone-engine = "^0.9.2"
+pyelftools = ">=0.28"
+gevent = ">=20.9.0"
+multiprocess = ">=0.70.12.2"
+windows-curses = { version = "^2.1.0", markers = "platform_system == 'Windows'" }
+pyyaml = "^6.0.1"
+python-fx = "*"
+questionary = "*"
+termcolor = "*"
+
+blake2b-py = { version = ">=0.1.2", optional = true }
+eth-keys = { version = ">=0.2.1", optional = true }
+eth-typing = { version = ">=2.2.0", optional = true }
+eth-utils = { version = ">=1.9.4", optional = true }
+eth_abi = { version = ">=2.1.1,<4", optional = true }
+lru-dict = { version = ">=1.1.6", optional = true }
+py-ecc = { version = ">=1.4.7", optional = true }
+rlp = { version = "*", optional = true }
+trie = { version = "*", optional = true }
+numpy = { version = "*", optional = true }
+rich = { version = "*", optional = true }
+cmd2 = { version = "*", optional = true }
+eth-hash = { extras = ["pycryptodome"], version = "*", optional = true }
+
+unicornafl = { version = "^2.0.0", markers = "platform_system != 'Windows'", optional = true }
+fuzzercorn = { version = "^0.0.1", markers = "platform_system == 'Windows'", optional = true }
```

Comment:
fuzzercorn should be installed on Linux only.

Reply:
Rebased and fixed.